homeassistant/components/media_player/sonos.py

```diff
@@ -193,6 +208,10 @@ def service_handle(service):
         DOMAIN, SERVICE_CLEAR_TIMER, service_handle,
         descriptions.get(SERVICE_CLEAR_TIMER), schema=SONOS_SCHEMA)
 
+    hass.services.register(
+        DOMAIN, SERVICE_UPDATE_ALARM_CLOCK, service_handle,
+        descriptions.get(SERVICE_UPDATE_ALARM_CLOCK), schema=SONOS_UPDATE_ALARM_CLOCK_SCHEMA)
```

Comment:
Could use SERVICE_UPDATE_ALARM instead which would make a lot of names shorter. What do others think?